tests/test_defects.py

```diff
@@ -348,8 +348,7 @@ def test_maskBlocks_blobs_one_side_good_less_than_threshold(self):
         """
 
         expectedDefects = [Box2I(corner=Point2I(60, 1), dimensions=Extent2I(1, 29)),
-                           Box2I(corner=Point2I(61, 2), dimensions=Extent2I(1, 12)),
-                           Box2I(corner=Point2I(62, 2), dimensions=Extent2I(1, 12))]
```

Comment:
The expected defects before these changes where individual boxes, even if they were adjacent. With the changes, the boxes get merged into fewer boxes that have a wider extension (for example, 2 pixels in one direction instead of 1).